docs/content/guides/recipes/Terraform/howto-private-registry/index.md

```diff
@@ -0,0 +1,72 @@
+---
+type: docs
+title: "How-To: Set up a private registry for your Terraform Radius Recipes"
+linkTitle: "Set up a private registry"
+description: "Learn how to configure your Radius Environment to leverage your private registry as storage for your custom Recipe templates"
+weight: 500
+categories: "How-To"
+tags: ["recipes", "terraform"]
+---
+
+This how-to guide will provide an overview of how to:
+
+- Configure a Radius Environment that leverages a [Radius secret store]({{< ref secretstore >}}) to register a [private Terraform registry](https://developer.hashicorp.com/terraform/registry/private) for your Recipe templates.
+
+### Prerequisites
+
+Before you get started, you'll need to make sure you have the following tools and resources:
+
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
```

Comment:
I'd add a step here for creating a PAT. You can link to the docs pages for some popular platforms like Azure DevOps, GitHub, and GitLab.